src/dfx/assets/new_project_node_files/package.json

```diff
@@ -8,7 +8,7 @@
     "prebuild": "npm run copy:types",
     "start": "webpack serve --mode development --env development",
     "prestart": "npm run copy:types",
-    "copy:types": "rsync -avr .dfx/$(echo ${DFX_NETWORK:-'**'})/canisters/** --exclude='assets/' --exclude='idl/' --exclude='*.wasm' --delete src/declarations"
+    "copy:types": "rsync -avr .dfx/$(echo $DFX_NETWORK | sed -e 's/[^0-9a-zA-Z_]/_/g' -e 's/^$/**/')/canisters/** --exclude='assets/' --exclude='idl/' --exclude='*.wasm' --delete src/declarations"
```

Comment:
How about we make this transformation in dfx, where we set DFX_NETWORK?

https://github.com/dfinity/sdk/blob/master/src/dfx/src/lib/builders/assets.rs#L277

Reply:
Good catch - should I set it all the way up here? https://github.com/dfinity/sdk/blob/ce91cf8ab0a3306950ec2425850511da6a7e3db0/src/dfx/src/lib/builders/mod.rs#L235

Reply:
Yeah, that sounds even better - but please check usages of that struct member to make sure.